packages/contentstack-audit/package.json

```diff
@@ -21,7 +21,7 @@
     "@contentstack/cli-command": "~1.2.16",
     "@contentstack/cli-utilities": "~1.6.0",
     "@oclif/plugin-help": "^5",
-    "@oclif/plugin-plugins": "^4.1.9",
+    "@oclif/plugin-plugins": "^5.0.0",
```

Comment:
@cs-raj have you validated if there are any breaking changes with this MAJOR version? As I can see, they released so many prereleases for `v5-x.x-beta` to test, I guess.

Reply:
I didn't find any breaking changes